### ISSUE TITLE
QML refactoring

### ISSUE THREAD

**Issue by athairus** [team-phoenix/Phoenix#185]
Organize the QML into folders:
```
Widgets/ (reusable components like MarqueeText)
MouseDrivenView/SidePanel/ (Everything that goes on the left)
MouseDrivenView/MainPanel/ (Gridview, etc)
GameView/
```

Use prefixes to keep everything on the filesystem "flat" while still having folders in Qt Creator.

You'll need to work around this new structure by importing the prefixes at the top of the file like any other import: `import "qrc:/Widgets"`, for example.

**Comment by athairus** [comment]
Scratch all that, I've followed what @Druage has already done and made more qrc files. If I also make qmldir files, the modules should be visible from Qt Creator.

**Comment by athairus** [comment]
I revisited the original proposal in `feature-pipeline` (#249) and it's much more pleasant to work with than any other approach I tried.

Close once #249 is merged.
